src/extension/metadata/src/main/java/org/geoserver/metadata/web/panel/ImportTemplatePanel.html

```diff
@@ -5,27 +5,26 @@
 <body>
 <wicket:panel>
 	<div wicket:id="importDialog"></div>
-    <ul>
-        <li>
-            <fieldset>
-                <legend>
-                    <span><wicket:message key="header"></wicket:message></span>
-                </legend>
-					<ul class="links-above-table">
-						<li><select wicket:id="metadataTemplate" /> <a href="#"
-							wicket:id="link"> <wicket:message key="actionImport" />
-						</a></li>
-						<li><a class="remove-link" href="#"
-							wicket:id="removeSelected"> <wicket:message
-									key="removeSelected" />
-						</a></li>
-					</ul>
-					<span wicket:id="linkTemplateFeedback"></span>
-					<div wicket:id="templatesPanel"></div>
-					<div wicket:id="noData"></div>
-				</fieldset>
-        </li>
-    </ul>
+
+	<fieldset>
+		<legend>
+			<span><wicket:message key="header"></wicket:message></span>
+		</legend>
+		<ul class="links-above-table">
+			<li><select wicket:id="metadataTemplate"/> <a href="#"
+																										wicket:id="link">
+				<wicket:message key="actionImport"/>
+			</a></li>
+			<li><a class="remove-link" href="#"
+						 wicket:id="removeSelected">
+				<wicket:message
+								key="removeSelected"/>
+			</a></li>
+		</ul>
+		<span wicket:id="linkTemplateFeedback"></span>
+		<div wicket:id="templatesPanel"></div>
+		<div wicket:id="noData"></div>
+	</fieldset>
```

Comment:
tabs vs spaces; do we care or have a expectation for html?

Reply:
@aaime is there any automatic formatting we should consider for the html pages.

Reply:
Eh, I've tried and failed with spotless, see the discussion here, dealing with XML files, but the HTML formatter also comes from the same Eclipse plugin (org.eclipse.wst):
https://github.com/diffplug/spotless/issues/1135

See also https://github.com/diffplug/spotless/tree/main/plugin-maven

Long story short, it should be doable but all QA configurations should go inside a Maven managed jar package... I'm not sure how to deal with the eventual circular dependency (the build would need it to run).

Reply:
And oh... the issue arised because I wanted to customize the formatting result. If you were so lucky as to find the default HTML output acceptable , then you could use it as-is.